src/inject/dynamic-theme/watch.ts

```diff
@@ -161,16 +161,21 @@ export function watchForStyleChanges(currentStyles: StyleElement[], update: (sty
 
     function handleAttributeMutations(mutations: MutationRecord[]) {
         const updatedStyles = new Set<StyleElement>();
+        const removedStyles = new Set<StyleElement>();
         mutations.forEach((m) => {
-            if (shouldManageStyle(m.target) && m.target.isConnected) {
-                updatedStyles.add(m.target as StyleElement);
+            if (m.target.isConnected) {
+                if (shouldManageStyle(m.target)) {
+                    updatedStyles.add(m.target as StyleElement);
+                } else if ((m.target as HTMLLinkElement).disabled && shouldManageStyle(m.target, {disableDisabled: true})) {
```

Comment:
Very nice, but ... is there a need in options.disableDisabled? Funny name BTW, should we disable disabled or enable disabled? 🤣 

Maybe replace with `} else if (m.target instanceof HTMLLinkElement && m.target.disabled) {`, it is not an expensive operation and code looks simpler.

Reply:
> Very nice, but ... is there a need in options.disableDisabled? Funny name BTW, should we disable disabled or enable disabled? ![](https://github.githubassets.com/images/icons/emoji/unicode/1f923.png)

`anPossibleCheckToExcludeDisableFromBeingTestedInTheCondition`